contracts/payloads/IGP7/mocks/PayloadIGP8Mock.sol

```diff
@@ -0,0 +1,226 @@
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+interface IGovernorBravo {
+    function _acceptAdmin() external;
+    function _setVotingDelay(uint newVotingDelay) external;
+    function _setVotingPeriod(uint newVotingPeriod) external;
+    function _acceptAdminOnTimelock() external;
+    function _setImplementation(address implementation_) external;
+    function propose(address[] memory targets, uint[] memory values, string[] memory signatures, bytes[] memory calldatas, string memory description) external returns (uint);
+    function admin() external view returns(address);
+    function pendingAdmin() external view returns(address);
+    function timelock() external view returns(address);
+    function votingDelay() external view returns(uint256);
+    function votingPeriod() external view returns(uint256);
+}
+
+interface ITimelock {
+    function acceptAdmin() external;
+    function setDelay(uint delay_) external;
+    function setPendingAdmin(address pendingAdmin_) external;
+    function queueTransaction(address target, uint value, string memory signature, bytes memory data, uint eta) external returns (bytes32);
+    function executeTransaction(address target, uint value, string memory signature, bytes memory data, uint eta) external payable returns (bytes memory);
+    function pendingAdmin() external view returns(address);
+    function admin() external view returns(address);
+    function delay() external view returns(uint256);
+}
+
+interface IInstaIndex {
+    function changeMaster(address _newMaster) external;
+    function updateMaster() external;
+    function master() external view returns(address);
+}
+
+interface ILite {
+    function setAdmin(address newAdmin) external;
+    function getAdmin() external view returns(address);
+    function updateSecondaryAuth(address secondaryAuth_) external;
+}
+
+interface IDSAV2 {
+    function cast(
+        string[] memory _targetNames,
+        bytes[] memory _datas,
+        address _origin
+    )
+    external
+    payable 
+    returns (bytes32);
+
+    function isAuth(address user) external view returns (bool);
+}
+
+contract PayloadIGP8Mock {
```

Comment:
nit: IGP7Mock?

Reply:
It's actually, IGP8Mock itself inorder to test that next proposal are also working fine after IGP7